grace/simulator/store_features.py

```diff
@@ -0,0 +1,74 @@
+import click
+import torch
+
+from pathlib import Path
+from grace.base import GraphAttrs
+from grace.io import write_graph
+from grace.io.image_dataset import ImageGraphDataset
+from grace.models.feature_extractor import FeatureExtractor
+
+
+# Define a click command to input the file name directly:
+@click.command(name="Graph Storage")
+@click.option(
+    "--data_path",
+    type=click.Path(exists=True),
+    help="Path to images and grace annotations",
```

Comment:
Maybe add that in this base path, the features will be saved as well.

Reply:
Now I see that we are overwriting the grace file, is that safe? Should it be an option in case the user wants to keep the original file?